modules/battery_manager/php/battery_manager.class.inc

```diff
@@ -0,0 +1,245 @@
+<?php
+/**
+ * This file contains the NDB_Menu_Filter_battery_manager class
+ *
+ * PHP Version 7
+ *
+ * @category Module
+ * @package  Battery_Manager
+ * @author   Victoria Foing <victoria.foing@mcin.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+namespace LORIS\battery_manager;
+
+/**
+ * Main class for battery manager module corresponding to /battery_manager/ URL
+ * Admin section of the LorisMenu.
+ *
+ * Displays a list of records in the test battery and control panel to search them
+ * Allows user to add, activate, and deactivate entries in the test battery
+ *
+ * PHP Version 7
+ *
+ * @category Module
+ * @package  Battery_Manager
+ * @author   Victoria Foing <victoria.foing@mcin.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+class Battery_Manager extends \NDB_Menu_Filter
+{
+    public $hasWritePermission = false;
+    public $AjaxModule         = true;
+    public $skipTemplate       = true;
+
+    /**
+     * Check user permissions
+     *
+     * @return bool
+     * @throws ConfigurationException
+     */
+    function _hasAccess()
```

Comment:
should include return type

Reply:
Still not resolved  
``` function _hasAccess() :Bool```

same for other function

Reply:
@victoriafoing in the function declaration.

i.e. "function _hasAccess() : bool"
in the case the function accept parameters:
function name(string $param1, int $param2) : string
you can see more at http://php.net/manual/en/migration70.new-features.php and http://php.net/manual/en/functions.arguments.php#functions.arguments.type-declaration